packages/babel-helper-define-polyfill-provider/src/visitors/usage.ts

```diff
@@ -8,7 +8,7 @@ function isRemoved(path: NodePath) {
   if (path.removed) return true;
   if (!path.parentPath) return false;
   if (path.listKey) {
-    if (!path.parentPath.node[path.listKey].includes(path.node)) return true;
+    if (!path.parentPath.node[path.listKey]?.includes(path.node)) return true;
```

Comment:
should put `?` after `node`.

```js
if (!path.parentPath.node?.[path.listKey].includes(path.node)) return true;
```

Reply:
```suggestion
    if (!path.parentPath.node?.[path.listKey]?.includes(path.node)) return true;
```

Let's have both, since it was needed by one of the other stack traces.